text/0000-rust-has-provenance.md

```diff
@@ -0,0 +1,330 @@
+- Feature Name: rust_has_provenance
+- Start Date: 2023-11-22
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
```

Comment:
Please fill-in these links and change the name of the file appropriately, so this can be merged =)

Reply:
Hu? I never did that for any of my RFCs that was merged before. Has the process changed?

Reply:
I filled the fields.

How does renaming the file to its final name work?

Reply:
Convention seems to be the PR number itself, in this case 3559

Reply:
I know. But usually that happens post-merge, I never was involved in that.

Reply:
Same, I've never done it myself.

Reply:
Huh. I thought that was always done by the author of the RFC

Reply:
Ah okay. :) I thought maybe there was some post-merge automation or whatever, didn't want to intefere with the standard process -- and didn't realize the process was updated.